Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -53,18 +80,71 @@ final class SettingsAccountViewController: UIViewController {
     _ = tableView
       |> settingsTableViewStyle
   }
+
+  func showCurrencyPickerCell(_ show: Bool) {
+    if show {
```

Comment:
Should we also add the `show == false` case here? Or remove the `show` parameter if it's not really needed.